main/management/commands/start.py

```diff
@@ -47,10 +49,17 @@ def handle(self, *args, **options):
             logging.error('Migrations could not be run, exiting.')
             sys.exit(1)
 
-        logging.info('Running setup')
+        if not Employee.objects.filter(username=os.environ['SUPER_USERNAME']).exists():
+            Employee.objects.create_superuser(username=os.environ['SUPER_USERNAME'],
+                                              password=os.environ['SUPER_PASSWORD'])
```

Comment:
Pet peeve, but consider using K&R style multiline breaks here, it's much more readable:

```py
Employee.objects.create_superuser(
    username=os.environ['SUPER_USERNAME'],
    password=os.environ['SUPER_PASSWORD'],
)
```

The benefit of this is that you don't have to read all this stuff that's super deeply indented.